rescript/ncbi_datasets.py

```diff
@@ -30,14 +29,14 @@
 
 def _get_assembly_descriptors(
         api_instance, assembly_levels, assembly_source, only_reference,
-        page_size, taxon, tax_exact_match
+        page_size, taxons, tax_exact_match
```

Comment:
```suggestion
        page_size, taxa, tax_exact_match
```
Why are you using sometimes the word taxons and sometimes taxa? Taxons is not correct or is there a reason?

Reply:
That's because the NCBI API uses `taxons`. But as you, correctly, pointed out that's not quite right so in my own code I'm using `taxa` instead (but perhaps not quite consistently) - I now changed it everywhere where not absolutely required.